dm/dm.go

```diff
@@ -71,13 +72,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
```

Comment:
May be able to determine if the file is a tar archive or not using the
library. Not sure about Windows.

On Friday, November 27, 2015, vaikas-google <notifications@github.com>
wrote:

> In dm/dm.go
> <https://github.com/kubernetes/deployment-manager/pull/131#discussion_r46075518>
> :
>
> > @@ -71,13 +72,15 @@ var usage = func() {
> >  	fmt.Fprintln(os.Stderr, "Flags:")
> >  	flag.PrintDefaults()
> >  	fmt.Fprintln(os.Stderr)
> > -	os.Exit(1)
> > +	fmt.Fprintln(os.Stderr, "--stdin requires a tar archive of input files and a top level file name.")
>
> I admit it's been a looong time, but does Windows come with tar? Can we
> look at the file and see if it's a tar file and if not, treat it as a
> single file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/deployment-manager/pull/131/files#r46075518>
> .
>


-- 
Jack Greenfield | Google